src/SSHLibrary/abstractclient.py

```diff
@@ -135,7 +135,7 @@ def login(self, username, password, delay=None, look_for_keys=False):
 
         :param bool look_for_keys: Whether the login method should look for
             available public keys for login. This will also enable ssh agent.
-            This option in ignored in Jython.
+            This option is ignored by Jython.
```

Comment:
Good type catch but "by Jython" is not that good either. Perhaps something like "This option is ignored when using Jython." would work better?

Reply:
Yep. Much better.